README.md

```diff
@@ -411,26 +411,25 @@ custom:
 
 ## Native Code Dependencies During Runtime
 
-Some Python packages require extra OS libraries (`*.so` files) at runtime. You need to manually include these files in the root directory of your Serverless package. The simplest way to do this is to commit the files to your repository:
+Some Python packages require extra OS libraries (`*.so` files) at runtime. You need to manually include these files in the root directory of your Serverless package. The simplest way to do this is to use the `dockerExtraFiles` option.
```

Comment:
Wouldn't you also need to have a `Dockerfile` that contains the interesting `.so` files? Eg, for the example below, it should be using a `Dockerfile` that installs MySQL.

Reply:
The next paragraph states "If you use the Dockerfile from the previous section", which does install MySQL. Would you instead prefer I copy and paste the same Dockerfile to this section? I'm fine either way, let me know what you prefer.